suite-native/discovery/src/utils.ts

```diff
@@ -1,24 +1,15 @@
 import { A, F } from '@mobily/ts-belt';
 
-import { NetworkCompatible, AccountType, NetworkSymbol } from '@suite-common/wallet-config';
+import { Network, NetworkSymbol } from '@suite-common/wallet-config';
 import { Account } from '@suite-common/wallet-types';
 
-const NORMAL_ACCOUNT_TYPE = 'normal';
-
-// network uses undefined for normal type, but account uses 'normal'
-const normalizedAccountType = (accountType?: AccountType) => accountType ?? NORMAL_ACCOUNT_TYPE;
-
 export const getNetworksWithUnfinishedDiscovery = (
-    enabledNetworks: readonly NetworkCompatible[],
+    enabledNetworks: readonly Network[],
     accounts: Account[],
     accountsLimit: number,
 ) =>
     enabledNetworks.filter(network => {
-        const networkAccountsOfType = accounts.filter(
-            account =>
-                account.symbol === network.symbol &&
-                account.accountType === normalizedAccountType(network.accountType),
-        );
```

Comment:
previously this filtered the duplicated `networks` only for the normal account. That's no longer needed, as they are no longer duplicated for each `accountType`